File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        int len=2*n;
        vector<int>ans(len,0);
        for(int i=0;i<len;i++){
            ans[i]=nums[i%n];
        }
        return ans;
    }
};

File: Data Structures & Algorithms/score-of-a-string/submission-1.java
class Solution {
    public int scoreOfString(String s) {
        int ans=0;
        int val=s.charAt(0);
        for(int i=1;i<s.length();i++){
    
         ans+=Math.abs(val-s.charAt(i)); 
         val=s.charAt(i);       
        }
        return ans;
        
    }
 
}